### ISSUE TITLE
[BUG] Footer not appearing on mobile browsers

### ISSUE THREAD

**Issue by sproules** [wet-boew/wet-boew#9323]
On a desktop browser, the footer block appears properly; however, on a mobile browser, the footer block doesn't appear at all.  While we're using WET version 4.0.27, this issue exists with the latest version as well.  In fact, this issue even exists on the WET site itself.  You can duplicate this issue by looking at the About page (https://wet-boew.github.io/v4.0-ci/index-en.html#about) on a mobile browser (I used Chrome on an Android phone) and the footer is not displayed.

**Comment by bo7owers** [comment]
I am not very familiar with WET but this behaviour is caused by the classes that are set into the footer (visible-sm visible-md visible-lg). Remove them and you will be able to see the footer in smaller displays, just make sure your footer is responsive. Hope this helps

**Comment by kodz1** [comment]
The footer is moved to the menu.

![2FF66938-0BEE-420B-8AB0-BC916BE8AD84](https://user-images.githubusercontent.com/23484460/165434107-40501006-7666-401b-b101-3bd00783e321.jpeg)

**Comment by sproules** [comment]
Thank you for the information about the footer appearing in the menu.  I still have an issue.  We have a central directory where the footer HTML resides so that all applications can use it.  Then we use jQuery (with ThymeLeaf) (see below) to replace a placeholder with the footer text.  This approach works fine on a desktop, but I don't see the footer in the menu in a mobile view (if I copy and paste the footer HTML into my page instead of using jQuery, I see the see the footer in the menu).

    <script th:inline="javascript">
        $(document).ready(function() {
            $("#edh_footer").load([[${commonFooterUrl}]]);
        });
    </script>

**Comment by kodz1** [comment]
Yeah the footer gets regenerated when on mobile. Secondly I confirm in a desktop when the window is resized if it is working as small screens (small laptops for example) switch to mobile menu.

**Comment by kodz1** [comment]
This page goes over it https://wet-boew.github.io/v4.0-ci/docs/ref/menu/menu-en.html

Might have to use this:
```
$( document ).on( "wb-ready.wb-menu", ".wb-menu", function( event, instance, settings ) {
});
```

**Comment by sproules** [comment]
I tried the kodz1's jQuery suggestion and it didn't work for mobile browsers.  I put alerts in the code and the alerts were triggered, but the footer didn't appear in the menu.  Instead, I'm reading the HTML footer files in Java controller code and passing the text to the ThymeLeaf template and that works well.

However, I do have a question.  One of our applications is used for KeyCloak login for all of our applications and therefore doesn't have a menu.  If there's no menu, will people see the footer in a mobile browser?

**Comment by kodz1** [comment]
As previous mentioned you could try removing any css classes from footer html that hide it like visible-sm visible-md visible-lg unsure if wb-navcurr affects this, but that would change the normal functionality

**Comment by sproules** [comment]
I removed the visible-sm, visible-md and visible-lg classes from the footer.  I got different results depending on how I called it.  If I used my old code (see below), I would see the footer text as both a footer and in the menu for mobile browsers.

        $(document).ready(function() {
            $("#edh_footer").load([[${commonFooterUrl}]]);
        });

If I used the other manner in which you suggested (see below), I see the footer text as a footer for mobile browsers.

        $( document ).on( "wb-ready.wb-menu", ".wb-menu", function( event, instance, settings ) {
            $("#edh_footer").load([[${commonFooterUrl}]]);
        });

Which is preferred?  I prefer not having the footer text duplicated in the footer and menu on mobile browsers.  If the footer text appears only in the footer on mobile browsers, would that affect screen readers?

**Comment by kodz1** [comment]
Sorry the late response, without a test environment it much harder to assist on this past the tips provided to try to configure this to your unique use case.

But for sure duplication would be a accessibility issue and confuse normal users.

###### I am just a volunteer on this GitHub project.
